test/fixtures/cpu_mmu.c

```diff
@@ -0,0 +1,9 @@
+#include "cpu_mmu.h"
+
+void cpu_mmu_setup(void) {
+    srand(time(NULL));
```

Comment:
We might have to rethink this, so we can achieve reproducible tests 🤔 But as far as I know, rand things are pretty fucked up in C/C++

Reply:
Do you have an idea? I wanted to set our memory address more random in contrast to hardcoding it in order to gain a broader test range regarding our memory

Reply:
I'm not sure tbh. I think that the first thing we could do is save a seed for the srand function. This should technically give us reproducibility. However, the problem with rand is that any other function can call it, such that we might miss a value when repeating a test, leading to a different result 🤔 

This sounds like a better alternative together with a seed: https://man7.org/linux/man-pages/man3/random.3.html